apps/backend/user/src/sendConfirmationEmail.ts

```diff
@@ -30,21 +29,22 @@ const sendConfirmationEmail = async (
   }
 
   const subject = "MartletPlace - Please confirm your email";
-  const token = "jwttoken";
+  const token = create_token({ userId: userId });
```

Comment:
This should have the optional second argument to restrict it to a specific route. See the auth.ts file in lib